src/source/video_source.js

```diff
@@ -91,6 +92,30 @@ class VideoSource extends ImageSource {
         });
     }
 
+    /**
+     * Pauses the video.
+     */
+    pause() {
+        this.video.pause();
```

Comment:
Needs `null` guard for `this.video`. It is `undefined` until the source finished loading